R/env-special.R

```diff
@@ -197,44 +197,51 @@ current_env <- function() {
 #' These functions are experimental and may not belong to the rlang
 #' package. Expect API changes.
 #'
-#' @param pkg The name of a package. If `NULL`, the surrounding
-#'   namespace is returned, or an error is issued if not called within
-#'   a namespace. If a function, the enclosure of that function is
-#'   checked.
+#' @param pkg The name of a package or an environment. If a function,
```

Comment:
Should we rename to `x`? 

I think a bulleted list would also make it more clear what the set of possible values is.

Reply:
Good idea.